tensordict/_td.py

```diff
@@ -308,6 +308,8 @@ def is_empty(self):
     def to_module(
         self,
         module,
+        *,
+        inplace: bool | None = None,
```

Comment:
Is `None` supported for backward compatibility ?

Reply:
We want to catch inplace=True when use_state_dict=False since this is not implemented.

By making it None, and given that inplace is True for state dict, we tell users: you explicitly asked for False but we can't make that happen.

Reply:
Thanks for explaining!